components/ingest-service/integration_test/suite_test.go

```diff
@@ -153,6 +162,17 @@ func (s *Suite) IngestNodes(nodes []iBackend.Node) {
 	s.RefreshIndices(mappings.NodeState.Index)
 }
 
+// IngestActions ingests a number of actions then refreshe the all the action indexes
+func (s *Suite) IngestActions(actions []iBackend.InternalChefAction) {
+	// Insert actions
+	for _, action := range actions {
+		s.ingest.InsertAction(context.Background(), action)
+	}
+
+	// Refresh Indices
+	s.RefreshIndices(mappings.Actions.Index + "-*")
```

Comment:
In the PR I merged the other day, I moved some of these index strings into reusable variables and the refresh actions on them into helper methods, because I was copying and pasting them repeatedly. This one is referred to at least twice in this file so it might deserve the same treatment?